src/ClimatePref/ReadData.jl

```diff
@@ -111,44 +114,15 @@ function readfile(file::String)
         AG.read!(bd, a);
     end;
     lat, long = size(a, 1), size(a, 2);
-    step = 360.0° / lat;
-    #step = 180.0° / long;
+    step_lat = (xmax - xmin) / lat;
+    step_long = (ymax - ymin) / long;
 
     world = AxisArray(a[:, long:-1:1],
-                           Axis{:latitude}((-180.0°+ step):step:(180.0°)),
-                           Axis{:longitude}((-90.0°):step:90.0°));
+                           Axis{:latitude}(xmin:step_lat:(xmax-step_lat)),
+                           Axis{:longitude}(ymin:step_long:(ymax-step_long)));
```

Comment:
There must be a floating point precision risk here that we lose the last element. Instead it should be:
```suggestion
                           Axis{:latitude}(xmin:step_lat:(xmax-step_lat/2.0)),
                           Axis{:longitude}(ymin:step_long:(ymax-step_long/2.0)));
```

Reply:
Again this is from `main` though!